integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts

```diff
@@ -9,14 +8,10 @@ context('Legal Sender Journey E2E', () => {
   beforeEach(() => {
     cy.task('reset')
     cy.task('stubAuthToken')
-    cy.task('stubRequestLink')
-    cy.task('stubVerifyLink')
     cy.task('stubCreateBarcode')
 
-    // Request a magic link and click it to sign in and land on the Find Recipient By Prison Number page
```

Comment:
This code can now be found in the `signInAsLegalSender` method, and called based on the feature flag.
`signInAsLegalSender` is a nice abstraction to signing on 👍